code/game/objects/items/toys.dm

```diff
@@ -1069,6 +1069,22 @@ obj/item/toy/cards/deck/syndicate/black
 	name = "tuxedo cat plushie"
 	icon_state = "tuxedocat"
 
+/obj/item/toy/plushie/voxplushie
+	name = "vox plushie"
+	desc = "A stitched-together vox, fresh from the skipjack. Press its belly to hear it skree!"
+	icon_state = "plushie_vox"
+	item_state = "plushie_vox"
+	var/cooldown = 0
+
+/obj/item/toy/plushie/voxplushie/attack_self(mob/user)

```

Comment:
There's a better way of doing cooldowns than this stuff using `spawn()` - just call `user.changeNext_move(CLICK_CD_MELEE)` at the top of this `attack_self` and get rid of the stuff involving cooldowns

Reply:
Finally had a chance to try this! Not sure if I'm missing anything, but after removing all cooldown code and adding the line suggested, nothing errored, but there's no cooldown happening at all. Anything else I should have added?

Reply:
Ahh, I think I'm mistaken, it's been a while since I looked at cooldown code haha 😄 

I think what you have is fine, sorry for the confusion

Reply:
No worries! Thanks for taking a look anyway, it's always appreciated! :D